masonite/snippets/scaffold/middleware.html

```diff
@@ -0,0 +1,17 @@
+""" {{ class }} Middleware """
+
+class {{ class }}Middleware:
+    """{{ class }} Middleware
+    """
+
+    def __init__(self, Request):
+        """ Inject Any Dependencies From The Service Container """
+        self.request = Request
+
+    def before(self):
+        """ Run This Middleware Before The Route Executes """
+        pass
+
+    def after(self):
+        """" Run This Middleware After The Route Executes """
```

Comment:
docstring is messed up. Has too many `""""` Also let's follow the docstrings I've been using in other files:

```python
    def after(self):
        """Run This Middleware After The Route Executes
        """
```

Reply:
@josephmancuso Made the changes as requested.